Library/Homebrew/dev-cmd/man.rb

```diff
@@ -118,6 +147,10 @@ def convert_man_page(markup, target)
       ronn.close_write
       ronn_output = ronn.read
       ronn_output.gsub!(%r{</var>`(?=[.!?,;:]?\s)}, "").gsub!(%r{</?var>}, "`") if format_flag == "--markdown"
+      unless format_flag == "--markdown"
+        ronn_output = ronn_output.gsub(/<code>/, "\\fB").gsub(%r{</code>}, "\\fR")
```

Comment:
Put each `.gsub` on a new line and ideally document what each one is doing.

Reply:
I've changed it so its self documented.